### ISSUE TITLE
Docs: Tailwind template example

### ISSUE THREAD

**Issue by silva96** [ddnexus/pagy#448]
### Before submitting...

- [X] I searched through the [Documentation](https://ddnexus.github.io/pagy/)
- [X] I searched through the [Issues](https://github.com/ddnexus/pagy/issues)
- [X] I searched through the [Q&A](https://github.com/ddnexus/pagy/discussions/categories/q-a)

### Description

Proposed Better Tailwind UI (responsive)

<img width="738" alt="image" src="https://user-images.githubusercontent.com/4019924/212802764-b08f7946-79d5-452a-8066-292623ee4db0.png">

Pagination partial:

```
<%= render partial: 'shared/pagination', locals: { records: @posts, pagy: @pagy } %>
```

```erb
<% if pagy.count > 10 %>
  <div class="flex items-center justify-between border-t border-gray-200 px-4 py-3 sm:px-6">
    <div class="flex flex-1 justify-between sm:hidden">
      <% if pagy.prev %>
        <a href="<%= pagy_url_for(pagy, pagy.prev) %>" class="relative inline-flex items-center rounded-md border border-gray-300 bg-white px-4 py-2 text-sm font-medium text-gray-700 hover:bg-gray-50">
          <%= pagy_t('pagy.nav.prev').html_safe %>
        </a>
      <% end %>
      <% if pagy.next %>
        <a href="<%= pagy_url_for(pagy, pagy.next) %>" class="relative ml-3 inline-flex items-center rounded-md border border-gray-300 bg-white px-4 py-2 text-sm font-medium text-gray-700 hover:bg-gray-50">
          <%= pagy_t('pagy.nav.next').html_safe %>
        </a>
      <% end %>
    </div>
    <div class="hidden sm:flex sm:flex-1 sm:items-center sm:justify-between">
      <div>
        <p class="text-sm text-gray-700">
          <%= t('pagination.showing') %>
          <span class="font-medium"><%= pagy.from %></span>
          <%= t('pagination.to') %>
          <span class="font-medium"><%= pagy.to %></span>
          <%= t('pagination.of') %> 
          <span class="font-medium"><%= pagy.count %></span> <%= t('pagination.results') %>
        </p>
      </div>
      <div>
        <%== pagy_nav(@pagy) %>
      </div>
    </div>
  </div>
<% end %>
```

```css

.pagy-nav.pagination {
  @apply isolate inline-flex -space-x-px rounded-md shadow-sm
}

.page.next a {
  @apply relative inline-flex items-center rounded-r-md border border-gray-300 bg-white px-2 py-2 text-sm font-medium text-gray-500 hover:bg-gray-50 focus:z-20
}

.page.prev a {
  @apply relative inline-flex items-center rounded-l-md border border-gray-300 bg-white px-2 py-2 text-sm font-medium text-gray-500 hover:bg-gray-50 focus:z-20;
}

.page.next.disabled {
  @apply relative inline-flex items-center rounded-r-md border border-gray-300 bg-slate-100 px-2 py-2 text-sm font-medium text-gray-500 hover:bg-gray-50 focus:z-20
}

.page.prev.disabled {
  @apply relative inline-flex items-center rounded-l-md border border-gray-300 bg-slate-100 px-2 py-2 text-sm font-medium text-gray-500 hover:bg-gray-50 focus:z-20;
}

.page a, .page.gap {
  @apply bg-white border-gray-300 text-gray-500 hover:bg-gray-50 relative inline-flex items-center border px-4 py-2 text-sm font-medium focus:z-20
}

.page.active {
  @apply z-10 border-indigo-500 bg-indigo-50 text-indigo-600 relative inline-flex items-center border px-4 py-2 text-sm font-medium focus:z-20
}
```

Works better with 

```ruby
# initializers/pagy.rb

Pagy::DEFAULT[:size] = [1, 1, 1, 1]
```

**Comment by benkoshy** [comment]
Thx for submitting. 

LGTM. 

I'll resolve on the 24th Jan 23.

**Comment by ddnexus** [comment]
We could add this in the tailwind page, as something like "Template example"

**Comment by benkoshy** [comment]
Closed in favour of: #449. If the PR is inadequate, please feel free to amend or re-open.

@silva96 thank you for your contribution and patience.
